webpage/src/app/(landing)/components/download-button.tsx

```diff
@@ -0,0 +1,77 @@
+'use client';
+
+import { BROWSERS, getBrowser } from '@/utils/getBrowser';
+import { Button, Link } from '@chakra-ui/react';
+import NextLink from 'next/link';
+import { ReactElement, useEffect, useState } from 'react';
+import { FaChrome, FaFirefox, FaSafari } from 'react-icons/fa6';
+
+interface DownloadButtonProps {}
```

Comment:
Esta bien que esta interfaz quede vacía?

Reply:
This shouldn’t cause any problems